### PR TITLE
AboutUS, ContactUs and Service Page Added

### PR DESCRIPTION
**Pull request by BHS-Harish**
**Issue #146 fixed**

### CODE REVIEW

**Comment by Kajol-Kumari** [comment]
@BHS-Harish  take the icons from here. These are the final icons that needs to be used in the app - https://drive.google.com/drive/folders/1rh-Zof6lhDa74RzbU247Gueqgxu56CyD?usp=sharing

**Comment by Kajol-Kumari** [comment]
@BHS-Harish  let's use https://drive.google.com/file/d/1J18PzwIauJ_UX5Hhq0ojdPI_-I50UBhK/view?usp=drive_link as home page's cover image and rest of the images as well you can pick from the drive link I shared, the name of the pictures says where it should be used

**Comment by BHS-Harish** [comment]
@RohitGupta1237  @Kajol-Kumari 

The Images were replaced... Please check it...

File: frontend/screens/Homepage.js [hunk 1]
```diff
@@ -112,15 +114,15 @@ const Homepage = ({route,navigation}) => {
...
+            to={"BrandService"}
           />
           <Depth1Frame2
             brands="Influencers"
             depth3Frame1={require("../assets/depth-3-frame-1.png")}
             propBackgroundColor="#fff"
             propFontFamily="Lexend-Regular"
             propColor="#121217"
-            to={"InfluencersList"}
+            to={"InfluencerService"}
```
Comment (Kajol-Kumari) [inline frontend/screens/Homepage.js:125]: why did we do this change? Also what's the purpose of `InfluencerService`??

**Pull request merged**